Source/UI/Hashtag/HashtagView.swift

```diff
@@ -0,0 +1,145 @@
+//
+//  HashtagView.swift
+//  Planetary
+//
+//  Created by Martin Dutra on 29/11/22.
+//  Copyright © 2022 Verse Communications Inc. All rights reserved.
+//
+
+import Analytics
+import CrashReporting
+import Logger
+import SwiftUI
+
+@MainActor
+struct HashtagView: View {
+    
+    var hashtag: Hashtag
+
+    private var strategy: FeedStrategy {
+        HashtagAlgorithm(hashtag: hashtag)
+    }
+
+    @State
+    var messages: [Message]?
+
+    @State
+    private var isLoadingMoreMessages = false
+
+    @State
+    private var offset = 0
+
+    @State
+    private var noMoreMessages = false
+
+    @EnvironmentObject
+    private var botRepository: BotRepository
+
+    var body: some View {
+        Group {
+            if let messages = messages {
+                ScrollView(.vertical, showsIndicators: false) {
+                    ZStack {
+                        LazyVStack {
+                            if messages.isEmpty {
+                                EmptyPostsView(description: Localized.Message.noPostsInHashtagDescription)
+                            } else {
+                                ForEach(messages) { message in
+                                    Button {
+                                        if let contact = message.content.contact {
```

Comment:
I don't think we need this `if` block in this case, because we shouldn't ever need to display contact messages here?

Reply:
👌